client/rest-high-level/src/test/java/org/elasticsearch/client/SearchIT.java

```diff
@@ -269,7 +269,7 @@ public void testSearchWithMatrixStats() throws IOException {
         assertEquals(5, matrixStats.getFieldCount("num2"));
         assertEquals(29d, matrixStats.getMean("num2"), 0d);
         assertEquals(330d, matrixStats.getVariance("num2"), 0d);
-        assertEquals(-0.13568039346585542, matrixStats.getSkewness("num2"), 0d);
+        assertEquals(-0.13568039346585542, matrixStats.getSkewness("num2"), 1.0e-16);
```

Comment:
This is puzzling indeed. It could be a bug, but it could also be due to differences in the JVM/hardware that causes the documents to be stored in a different order in the index. Then at search time, aggs collect documents in a different order as well and we get different rounding errors?